cos-fleetshard-it/cucumber/src/main/java/org/bf2/cos/fleetshard/it/cucumber/ConnectorContext.java

```diff
@@ -124,12 +124,7 @@ private Map<String, Object> getPlaceholders() {
             }
             if (null != connector().getMetadata()) {
                 placeholders.put(COS_MANAGED_CONNECTOR_NAME, connector().getMetadata().getName());
-
-                if (connector().getMetadata().getLabels().containsKey(LABEL_UOW)) {
-                    placeholders.put(
-                        COS_DEPLOYMENT_UOW,
-                        connector().getMetadata().getLabels().get(LABEL_UOW));
-                }
+                placeholders.put(COS_TRAITS_CHECKSUM, Resources.computeTraitsChecksum(connector()));
```

Comment:
Here you are calculating the checksum of the `managedConnector` that is 0 and then you compare that with the one calculated for the `kameletBinding`

Reply:
yeah, that's wrong

Reply:
fixed